src/main/java/com/database/data/Company.java

```diff
@@ -0,0 +1,17 @@
+package com.database.data;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+
+@Data
+@AllArgsConstructor
+@RequiredArgsConstructor
+@ToString
```

Comment:
Annotation @ToString is redundant as a method toString() is generated when you use annotation @Data